controllers/articles.controller.js

```diff
@@ -11,4 +11,19 @@ exports.getAllArticles = (req, res, next) => {
     selectAllArticles().then((articles) => {
         res.status(200).send({ articles });
     }).catch(next);
+};
+
+exports.patchArticle = (req, res, next) => {
+    const articleId = req.params.article_id;
+    const { inc_votes } = req.body;
+
+    const promises = [updateArticle(articleId, inc_votes)];
```

Comment:
Remember, whenever we make a request that finds nothing, it will return an empty row, so we can use that to reject our promise here. If the article_id is missing i.e. null, psql will throw an error that we can make use of. This logic should be happening in the model to allow us to catch the error correctly.